api/http/handler/endpoints/filter.go

```diff
@@ -19,6 +19,8 @@ import (
 	"github.com/pkg/errors"
 )
 
+const edgeIntervalUseDefault = -1
```

Comment:
I think this can be inside the getShortestAsyncInterval fn. it's not needed elsewhere.

or, as this is a duplicate of https://github.com/portainer/portainer-ee/blob/840535c32303d3df4f98220021b5528753f08f60/api/http/handler/endpointedge/endpointedge_async.go#L29 maybe we can move into the edge module to be shared?